x-pack/plugin/security/build.gradle

```diff
@@ -87,6 +148,8 @@ dependencyLicenses {
 licenseHeaders {
     // This class was sourced from apache directory studio for some microsoft-specific logic
     excludes << 'org/elasticsearch/xpack/security/authc/ldap/ActiveDirectorySIDUtil.java'
+    // Test dependency for Kerberos testing with Apache DS
```

Comment:
this isn't a dependency as we are sourcing an entire class. The comment here should be like the comment above